packages/integration/lib/src/utils/media/video/video_thumbnail.dart

```diff
@@ -1,52 +1,27 @@
-import 'dart:async';
-
-import 'package:flutter/services.dart';
-import 'package:video_thumbnail/video_thumbnail.dart';
+import 'package:flutter/foundation.dart';
+import 'package:video_compress/video_compress.dart';
 
 class LMFeedVideoThumbnail {
-  /// Generates a thumbnail file under specified thumbnail folder or given full path and name which matches expected ext.
-  /// The video can be a local video file, or an URL repreents iOS or Android native supported video format.
-  /// If the thumbnailPath is ommited or null, a thumbnail image file will be created under the same folder as the video file.
-  /// Specify the maximum height or width for the thumbnail or 0 for same resolution as the original video.
-  /// The lower quality value creates lower quality of the thumbnail image, but it gets ignored for PNG format.
-  static Future<String?> thumbnailFile(
-      {required String video,
-      Map<String, String>? headers,
-      String? thumbnailPath,
-      ImageFormat imageFormat = ImageFormat.PNG,
-      int maxHeight = 0,
-      int maxWidth = 0,
-      int timeMs = 0,
-      int quality = 10}) async {
-    return VideoThumbnail.thumbnailFile(
-        video: video,
-        thumbnailPath: thumbnailPath,
-        imageFormat: imageFormat,
-        maxHeight: maxHeight,
-        maxWidth: maxWidth,
-        timeMs: timeMs,
-        quality: quality);
-  }
-
-  /// Generates a thumbnail image data in memory as UInt8List, it can be easily used by Image.memory(...).
-  /// The video can be a local video file, or an URL repreents iOS or Android native supported video format.
-  /// Specify the maximum height or width for the thumbnail or 0 for same resolution as the original video.
-  /// The lower quality value creates lower quality of the thumbnail image, but it gets ignored for PNG format.
-  static Future<Uint8List?> thumbnailData({
-    required String video,
-    Map<String, String>? headers,
-    ImageFormat imageFormat = ImageFormat.PNG,
-    int maxHeight = 0,
-    int maxWidth = 0,
-    int timeMs = 0,
+  /// Get thumbnail of video file
+  /// [path] : path of video file
+  /// [position] : position of thumbnail in video
+  /// [quality] : quality of thumbnail
+  /// Returns path of thumbnail file
+  static Future<String?> thumbnailFile({
+    required String path,
+    int position = 0,
     int quality = 10,
   }) async {
-    return VideoThumbnail.thumbnailData(
-        video: video,
-        imageFormat: imageFormat,
-        maxHeight: maxHeight,
-        maxWidth: maxWidth,
-        timeMs: timeMs,
-        quality: quality);
+    try {
+      final thumbnail = await VideoCompress.getFileThumbnail(
```

Comment:
Do we have support for height or width as well? or it will use the same dimension as video? This might be needed in Media Handling aspect ratio flow.

Reply:
it will take the same dimension.